### ISSUE TITLE
I am getting Could not find artifact io.quarkus:quarkus-logging-loki:jar:919-SNAPSHOT

### ISSUE THREAD

**Issue by abvijaykumar** [pilhuhn/quakus-logging-loki#1]
Hi, I am getting 
Could not find artifact io.quarkus:quarkus-logging-loki:jar:919-SNAPSHOT

when I use 

    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-logging-loki</artifactId>
      <version>919-SNAPSHOT</version>
    </dependency>

am I using the right version?

**Comment by pilhuhn** [comment]
Did you first build the extension as described in the REAME? Also this PoC is pretty old, so may no longer be working for an up to date version of Quarkus.

**Comment by abvijaykumar** [comment]
No, I don't find those instructions...anyway I am thinking not to use it, since u r saying its old. Thanks

**Comment by abvijaykumar** [comment]
what is ur recommendation to use Loki with my Quarkus application?

**Comment by pilhuhn** [comment]
The instructions are here https://github.com/pilhuhn/quakus-logging-loki#building

I think you can just log to stdout (default) and then use Promtail to collect the logs. I don't know the details of this though.
